addons/point_of_sale/static/src/app/debug/debug_widget.js

```diff
@@ -92,80 +90,89 @@ export class DebugWidget extends Component {
         this.state.barcodeInput = ean;
         await this.barcodeReader.scan(ean);
     }
-    async deleteOrders() {
-        this.dialog.add(ConfirmationDialog, {
-            title: _t("Delete Paid Orders?"),
-            body: _t(
-                "This operation will permanently destroy all paid orders from the local storage. You will lose all the data. This operation cannot be undone."
-            ),
-            confirm: () => {
-                // this.pos.db.remove_all_orders();
-                this.data.resetUnsyncQueue();
-            },
-        });
+    _createBlob(contents) {
+        if (typeof contents !== "string") {
+            contents = JSON.stringify(contents, null, 2);
+        }
+        return new Blob([contents]);
     }
-    async deleteUnpaidOrders() {
+    deleteOrders(paid = false) {
```

Comment:
unifying the 2 delete functions is awesome, but could you please make the api more explicit?
seeing `deleteOrders(true)` would not suggest in any way that i am only deleting the `paid` orders.
Something like this would be more explicit
```suggestion
    deleteOrders({ paid = true } = {}) {
```